src/helperFunctions/web_interface.py

```diff
@@ -16,12 +17,17 @@
     'ŔŕŘřẞßŚśŜŝŞşŠšȘșŤťŢţÞþȚțÜüÙùÚúÛûŰűŨũŲųŮůŪūŴŵÝýŸÿŶŷŹźŽžŻż'
 )
 
-BS_PRIMARY = '#007bff'
-BS_SECONDARY = '#6c757d'
 
+def get_color_list(number, limit=10):
+    color_map = cm.get_cmap('rainbow')
+    color_list = [colors.rgb2hex(color_map(i)) for i in range(32, 256, 22)]
+    return color_list[:min(number, limit)]
 
-def get_color_list(number, limit=15):
-    return ([BS_PRIMARY, BS_SECONDARY, ] * 8)[:number if number <= limit else limit]
+
+def get_alternating_color_list(number, limit=10):
```

Comment:
Maybe in this case, a comment actually makes sense. Without context, the magic number 8 (7) is rather confusing.